mne/epochs.py

```diff
@@ -37,6 +37,7 @@
 from .channels.channels import (ContainsMixin, UpdateChannelsMixin,
                                 SetChannelsMixin, InterpolationMixin)
 from .filter import resample, detrend, FilterMixin
+from .decoding.mixin import VectorizerMixin
```

Comment:
Can you try deleting this import (and the subclassing below) to see if it fixes the problem? That will tell us if it's a circular import problem or not.